test/api/api.js

```diff
@@ -1,212 +1,232 @@
+/* eslint-env qunit */
+/* global videojs */
 /**
  * These tests run on the minified, window.videojs and ensure the needed
  * APIs still exist
  */
+import document from 'global/document';
+import window from 'global/window';
 
-(function(){
+QUnit.module('Player API');
 
-q.module('Player API');
-
-test('videojs should exist on the window', function() {
-  ok(window.videojs, 'videojs exists on the window');
+QUnit.test('videojs should exist on the window', function() {
+  QUnit.ok(window.videojs, 'videojs exists on the window');
 });
 
-test('should be able to access expected player API methods', function() {
-  var player = videojs.getComponent('Player').prototype;
+QUnit.test('should be able to access expected player API methods', function() {
+  const player = videojs.getComponent('Player').prototype;
 
   // Native HTML5 Methods
-  ok(player.error, 'error exists');
-  ok(player.src, 'src exists');
-  ok(player.currentSrc, 'currentSrc exists');
-  ok(player.buffered, 'buffered exists');
-  ok(player.load, 'load exists');
-  ok(player.seeking, 'seeking exists');
-  ok(player.currentTime, 'currentTime exists');
-  ok(player.duration, 'duration exists');
-  ok(player.paused, 'paused exists');
-  ok(player.ended, 'ended exists');
-  ok(player.autoplay, 'autoplay exists');
-  ok(player.loop, 'loop exists');
-  ok(player.play , 'play exists');
-  ok(player.pause , 'pause exists');
-  ok(player.controls, 'controls exists');
-  ok(player.volume, 'volume exists');
-  ok(player.muted, 'muted exists');
-  ok(player.width, 'width exists');
-  ok(player.height, 'height exists');
-  ok(player.poster, 'poster exists');
-  ok(player.textTracks, 'textTracks exists');
-  ok(player.requestFullscreen, 'requestFullscreen exists');
-  ok(player.exitFullscreen, 'exitFullscreen exists');
-  ok(player.playbackRate, 'playbackRate exists');
-  ok(player.networkState, 'networkState exists');
-  ok(player.readyState, 'readyState exists');
+  QUnit.ok(player.error, 'error exists');
```

Comment:
I would've been _sooooo_ tempted to refactor this into a loop. I'm glad you resisted the urge! 😄